index.rst

```diff
@@ -166,30 +172,33 @@ One can also look at the relationship between Y-hexapod motion and X-Coma. One w
 .. figure:: /_static/X-coma_as_x_fxn_of_Y-displacement.jpg
     :name: X-Coma plot
 
-    This can be used in term FIXME of the sensitivity matrix, but is omitted as it's significantly smaller than the
+    This could be used in the sensitivity matrix, but is omitted as it's significantly smaller than the
     others and probably within the measurement errors due to turbulence.
 
 
 Applying the Fits to the Sensitivity Matrix
 ===========================================
 
-The sensitivity matrix is currently only in latiss_cwfs_align.py script. In order to use another matrix, either that
+The sensitivity matrix is currently only in
+`latiss_cwfs_align.py<https://github.com/lsst-ts/ts_externalscripts/blob/develop/python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py>`_
```

Comment:
Malformed link. I think it is because of a missing space before `<`

```
`latiss_cwfs_align.py <https://github.com/lsst-ts/ts_externalscripts/blob/develop/python/lsst/ts/externalscripts/auxtel/latiss_cwfs_align.py>`_
```